apps/design/frontend/src/tabs.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 import { useHistory, useLocation } from 'react-router-dom';
 import { Row } from './layout';
 
-interface Tab extends Route {}
+type Tab = Route;
```

Comment:
Since extending a type with no additional properties is (almost) the same as a type alias, they now require this style for clarity.